connect.js

```diff
@@ -137,6 +137,10 @@ const wrap = (Wrapped, module, epics, logger, options = {}) => {
           if (resource.shouldReset()) {
             resource.reset();
           }
+
+          if (!resource.isVisible()) {
```

Comment:
Could you please clarify what `isVisible` stands for?

Reply:
@VictorSoroka1 the `isVisible` is used to track if the given resource belongs to a currently visible (mounted) component. The same resource could be reused between multiple instances of the same component. Every time given instance is mounted we mark the resource as visible here:

https://github.com/folio-org/stripes-connect/blob/master/connect.js#L118

And when the component unmounts we mark is as invisible:

https://github.com/folio-org/stripes-connect/blob/master/connect.js#L135

At the end this is just a simple counter:

https://github.com/folio-org/stripes-connect/blob/master/OkapiResource.js#L60

When the `counter  === 0` we know that the given resource is currently not used by any component.

This idea was originally used when @skomorokh and I were working on the refresh functionality:

https://github.com/folio-org/stripes-connect/blob/master/epics/refreshEpic.js#L6

and we wanted to make sure the refresh doesn't execute too often (for unused resources).

Reply:
Thank you @mkuklis.